stats/scatterseries.cpp

```diff
@@ -126,6 +127,17 @@ std::vector<int> ScatterSeries::getItemsUnderMouse(const QPointF &point) const
 	return res;
 }
 
+void ScatterSeries::selectItemsUnderMouse(const QPointF &point)
+{
+	std::vector<struct dive *> selected;
+	struct dive *new_current;
+
+	for(int idx: getItemsUnderMouse(point)) {
+		selected.emplace_back(new_current = items[idx].d);
```

Comment:
Two small comments on this function:

1) This should read `push_back(...)`, not `emplace_back(...)`. the former adds a value to the vector, the later constructs an object in place. `emplace_back()` makes space for the object and then calls the constructor with the given parameters on the unitialized space. A pointer clearly has value semantics, so `push_back` is preferred, even though the latter works.

2) It appears to me that if getItemsUnderMouse() returns no element, then `new_current` is uninitialized - I wonder why the compiler doesn't complain about that...?